tests/unit/lib/foxbox/foxbox_test.js

```diff
@@ -0,0 +1,84 @@
+import Foxbox from 'js/lib/foxbox/foxbox';
+
+describe('Foxbox >', function() {
+  let foxbox;
+  const singleBox = [{
+    public_ip: '1.1.1.1',
+    client: 'abc',
+    message: JSON.stringify({
+      local_origin: 'https://local.abc.box.knilxof.org:3000',
+      tunnel_origin: 'null',
+    }),
+    timestamp: Math.floor(Date.now() / 1000),
+  }];
+  const multiBox = singleBox.concat([
+    {
+      public_ip: '2.2.2.2',
+      client: 'def',
+      message: JSON.stringify({
+        local_origin: 'http://local.def.box.knilxof.org:3000',
+        tunnel_origin: 'null',
+      }),
+      timestamp: Math.floor(Date.now() / 1000),
+    },
+  ]);
+
+  function getMockNet(single = true) {
```

Comment:
>  in its own file,

Honestly I don't really like mocks in a separate files like some used to do in gaia. Imo using sinon is a way more flexible and easy....

Moreover how do you feel about something more sinon-y and unit-testy :) Like this

```
 let netStub;

  beforeEach(function() {
    netStub = sinon.stub({
      init: () => {},
      fetchJSON: () => {}
    });
  });
  
  it/before/any-block-you-need(..., function() {
    netStub.fetchJSON.withArgs(...guard yourself with correct args.).returns(multiBox);
    
    // or 

    netStub.fetchJSON.withArgs(...guard yourself with correct args.).returns(singleBox);
  });
```